cli/cmd/runtime/start.go

```diff
@@ -125,7 +125,13 @@ func StartCmd(cliCfg *config.Config) *cobra.Command {
 				QueryCacheSizeBytes: conf.QueryCacheSizeBytes,
 				AllowHostAccess:     conf.AllowHostAccess,
 				SafeSourceRefresh:   conf.SafeSourceRefresh,
-				GlobalDrivers:       parse(conf.GlobalDrivers),
+				SystemConnectors: []*runtimev1.Connector{
+					{
+						Type:   conf.MetastoreDriver,
+						Name:   "metastore",
+						Config: map[string]string{"dsn": conf.MetastoreURL},
+					},
+				},
```

Comment:
Should update `runtime.Options.MetastoreDriver` to be `"metastore"` now, right? And probably rename `runtime.Options.MetastoreDriver` to `runtime.Options.MetastoreConnector`?